### PR TITLE
Updated to InfoBar colors/resources

### PR DESCRIPTION
**Pull request by teaP**
- Update background colors to common colors
- Remove close button border in rest state
- Add 1px border around the whole thing + rounded corners
- Update InfoBarHyperlinkForegroundColor to AccentTextFillColorPrimaryBrush
- Update InfoBarActionButtonCornerRadius to be ControlCornerRadius

### CODE REVIEW

**Comment by mdtauk** [comment]
Any chance of adding images of the changes @teaP

**Comment by teaP** [comment]
image

**Comment by mdtauk** [comment]
> > Also does this cover issue #3744 and my comment [#3744 (comment)](https://github.com/microsoft/microsoft-ui-xaml/issues/3744#issuecomment-756852744) ?
> 
> image
> 
> Yes, I think it does cover that issue since there's a border now.

Thank you, the border does help a bit.  The rounded corners may look less polished without some padding around the Information Bar in the UI however.

image

The updated ContentDialog seems to be using a White colour, with the bottom part using the new SolidBackgroundColor Fill - so could the InfoBar use White ``SolidBackgroundFillColorQuarternaryBrush``, to "pop" a little better?

**Comment by teaP** [comment]
> The updated ContentDialog seems to be using a White colour, with the bottom part using the new SolidBackgroundColor Fill - so could the InfoBar use White `SolidBackgroundFillColorQuarternaryBrush`, to "pop" a little better?

Sorry, but I'm not the one picking the colors. :)

**Comment by mdtauk** [comment]
> > The updated ContentDialog seems to be using a White colour, with the bottom part using the new SolidBackgroundColor Fill - so could the InfoBar use White `SolidBackgroundFillColorQuarternaryBrush`, to "pop" a little better?
> 
> Sorry, but I'm not the one picking the colors. :)

I am aware of that, I am posting the suggestion so that the reviewers can consider it.  I know you are updating values specified by the Design team.

The fact that this is an information bar, should mean it is designed to stand out - but Informational means it should not carry any semantic meaning, by using the Green, Red or Yellow colours.

File: dev/InfoBar/InfoBar.xaml [hunk 1]
```diff
@@ -102,6 +104,7 @@
                             HorizontalAlignment="Stretch"
                             MinHeight="{ThemeResource InfoBarMinHeight}"
                             Background="{TemplateBinding Background}"
+                            contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
```
Comment (marksfoster) [inline dev/InfoBar/InfoBar.xaml:107]: This seems redundant as the parent (`x:Name="ContentRoot"`) of this grid already has it defined.

File: dev/InfoBar/InfoBar_themeresources.xaml [hunk 2]
```diff
@@ -7,10 +7,10 @@
...
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Light">
-            <SolidColorBrush x:Key="InfoBarErrorSeverityBackgroundBrush" Color="#FDE7E9"/>
-            <SolidColorBrush x:Key="InfoBarWarningSeverityBackgroundBrush" Color="#FFF4CE"/>
-            <SolidColorBrush x:Key="InfoBarSuccessSeverityBackgroundBrush" Color="#DFF6DD"/>
-            <SolidColorBrush x:Key="InfoBarInformationalSeverityBackgroundBrush" Color="#F2F2F2"/>
+            <StaticResource x:Key="InfoBarErrorSeverityBackgroundBrush" ResourceKey="SystemFillColorCriticalBackgroundBrush"/>
+            <StaticResource x:Key="InfoBarWarningSeverityBackgroundBrush" ResourceKey="SystemFillColorCautionBackgroundBrush"/>
+            <StaticResource x:Key="InfoBarSuccessSeverityBackgroundBrush" ResourceKey="SystemFillColorSuccessBackgroundBrush"/>
+            <StaticResource x:Key="InfoBarInformationalSeverityBackgroundBrush" ResourceKey="SystemFillColorAttentionBackgroundBrush"/>
```
Comment (marksfoster) [inline dev/InfoBar/InfoBar_themeresources.xaml:13]: Nit: to match global WinUI style syntax, add single space before all closing tags

Comment (chingucoding) [reply dev/InfoBar/InfoBar_themeresources.xaml:13]: That is a pattern? I don't think there is a strong favor for either of those spacing styles.

Comment (marksfoster) [reply dev/InfoBar/InfoBar_themeresources.xaml:13]: When I look at generic.xaml, almost all closing tags have a leading space which makes me think it was a deliberate decision to make it the style.

Comment (chingucoding) [reply dev/InfoBar/InfoBar_themeresources.xaml:13]: I guess this wasn't brought forward to the WinUI code base then.

File: dev/InfoBar/InfoBar.xaml (hunk 1 above)
Comment (teaP) [reply dev/InfoBar/InfoBar.xaml:107]: You know, I thought that too at first! But if I don't set it here, setting the Background will fill in the corners.

File: dev/InfoBar/InfoBar_themeresources.xaml (hunk 2 above)
Comment (teaP) [reply dev/InfoBar/InfoBar_themeresources.xaml]: It's kind of random. I see it both ways. But for this file... sure. :)

File: dev/InfoBar/InfoBar_themeresources.xaml [hunk 3]
```diff
@@ -19,17 +19,17 @@
 
             <StaticResource x:Key="InfoBarTitleForeground" ResourceKey="TextFillColorPrimaryBrush" />
             <StaticResource x:Key="InfoBarMessageForeground" ResourceKey="TextFillColorPrimaryBrush" />
-            <StaticResource x:Key="InfoBarHyperlinkButtonForeground" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="InfoBarHyperlinkButtonForeground" ResourceKey="AccentTextFillColorPrimaryBrush" />
 
-            <StaticResource x:Key="InfoBarBorderBrush" ResourceKey="GhostFillColorTransparentBrush" />
-            <Thickness x:Key="InfoBarBorderThickness">0</Thickness>
+            <StaticResource x:Key="InfoBarBorderBrush" ResourceKey="CardStrokeColorDefault" />
```
Comment (marksfoster) [inline dev/InfoBar/InfoBar_themeresources.xaml:24]: Change to: CardStrokeColorDefaultBrush

File: dev/InfoBar/InfoBar_themeresources.xaml [hunk 4]
```diff
@@ -38,17 +38,17 @@
...
```
Comment (marksfoster) [inline dev/InfoBar/InfoBar_themeresources.xaml:43]: Change to: CardStrokeColorDefaultBrush

File: dev/InfoBar/InfoBar_themeresources.xaml (hunk 3 above)
Comment (teaP) [reply dev/InfoBar/InfoBar_themeresources.xaml]: Good catch, thanks!

**Review by marksfoster — APPROVED** [review]

**Comment by ranjeshj** [comment]
> > > The updated ContentDialog seems to be using a White colour, with the bottom part using the new SolidBackgroundColor Fill - so could the InfoBar use White `SolidBackgroundFillColorQuarternaryBrush`, to "pop" a little better?
> > 
> > 
> > Sorry, but I'm not the one picking the colors. :)
> 
> I am aware of that, I am posting the suggestion so that the reviewers can consider it. I know you are updating values specified by the Design team.
> 
> The fact that this is an information bar, should mean it is designed to stand out - but Informational means it should not carry any semantic meaning, by using the Green, Red or Yellow colours.

@YuliKl and @gabbybilka to follow up on this.

**Comment by Shomnipotence** [comment]
Is it just me that feels too much like macOS?

**Pull request merged**